TESTING.md

```diff
@@ -1,41 +1,28 @@
 # APM Server-Testing
 
 ## Automated Testing
-To run the full testsuite of APM Server, run:
+The tests are built on top of the [Beats Test Framework](https://github.com/elastic/beats/blob/master/docs/devguide/testing.asciidoc), where you can find a detailed description on how to run the Testsuite.
```

Comment:
Say "test suite" (two words with no caps) instead of "testsuite" unless you are referring to the command itself. Apply this comment globally.